mcp-spring/mcp-spring-webflux/src/test/java/io/modelcontextprotocol/WebFluxSseIntegrationTests.java

```diff
@@ -396,7 +398,7 @@ void testToolCallSuccess(String clientType) {
 					// perform a blocking call to a remote service
 					String response = RestClient.create()
 						.get()
-						.uri("https://github.com/modelcontextprotocol/specification/blob/main/README.md")
+						.uri("https://raw.githubusercontent.com/modelcontextprotocol/modelcontextprotocol/refs/heads/main/README.md")
```

Comment:
Would be nice if we ran something locally instead of reaching out to a real url.